frontend/src/pages/Dashboard/Dashboard.tsx

```diff
@@ -60,30 +278,102 @@ const Dashboard: FunctionComponent<DashboardParams> = ({
+  }, []);
+
   useEffect(() => {
     if (!currentUser) {
       getUser({});
     }
   }, [currentUser, getUser]);
+
   if (!currentUser) {
     return <h1>Loading</h1>;
   }
+
   return (
-    <Layout>
+    <Layout shouldAddPadding={false}>
       <>
-        Dashboard
-        <SignOutButton onClick={signOut}>sign out</SignOutButton>
-        <PurpleButton
-          text={getTitleOfButton(currentUser)}
-          onClick={() =>
-            history.push(
-              currentUser.onAssessment
-                ? "/assessments"
-                : "/interventions"
-            )
-          }
-          disabled={currentUser.sessionId === 9}
-        />
+        <DashboardContainer>
+          <MenuContainer>
+            <ProfilePicture
+              src={
+                "https://firebasestorage.googleapis.com/v0/b/vocab-buddy-53eca.appspot.com/o/dajin.png?alt=media&token=933c72b9-afaf-407b-b978-bfd2c3b4e155"
+              }
+            />
+            <TitleText>hi name!</TitleText>
+            <MenuButtonContainer></MenuButtonContainer>
+
+            {screenWidth > 600 && (
+              <SignOutButton onClick={signOut}>log out</SignOutButton>
+            )}
+          </MenuContainer>
+
+          <SessionContainer>
+            <TitleText>next session</TitleText>
+            <NextSessionButton
+              text={getTitleOfButton(currentUser)}
+              onClick={() =>
+                history.push(
+                  currentUser.onAssessment
+                    ? "/assessments"
+                    : "/interventions"
+                )
+              }
+              disabled={currentUser.sessionId === 9}
+            />
+
+            <TitleText>list of sessions</TitleText>
+          </SessionContainer>
+
+          <WeekProgressContainer>
+            <TitleText>this week</TitleText>
+
+            <WeekContainer>
+              <p>stars!</p>
+            </WeekContainer>
+
+            <TitleText>your progress</TitleText>
+            <ProgressStatsContainer>
+              <ProgressBox>
+                <ProgressStatNumber>14</ProgressStatNumber>
+                <ProgressStatDescription>
+                  day streak
+                </ProgressStatDescription>
+              </ProgressBox>
+
+              <ProgressBox>
+                <ProgressStatNumber>25</ProgressStatNumber>
+                <ProgressStatDescription>
+                  words learned
+                </ProgressStatDescription>
+              </ProgressBox>
+
+              <ProgressBox>
+                <ProgressStatNumber>4</ProgressStatNumber>
+                <ProgressStatDescription>
+                  assessments completed
+                </ProgressStatDescription>
+              </ProgressBox>
+
+              <ProgressBox>
+                <ProgressStatNumber>3</ProgressStatNumber>
+                <ProgressStatDescription>
+                  interventions completed
+                </ProgressStatDescription>
+              </ProgressBox>
```

Comment:
I think we should make a separate component for this, since it is used everywhere. It can be just a function in this file, but should probably be extracted.